.eslintrc.json

```diff
@@ -21,6 +21,14 @@
 		"jsdoc"
 	],
 	"rules": {
+    "@typescript-eslint/consistent-type-imports": [
```

Comment:
Please indent this one level.

Reply:
:+1: 

I formatted the whole document.